File: compRasp/fib.py
# A01732610 - Dafne Vania Peña Cortés

print("Los primeros 19 números de la serie Fibonacci son:")
n = 18

n_minus_two = 0
print(f"{n_minus_two}, ", end='', flush=True)
n_minus_one = 1
print(f"{n_minus_one}, ", end='', flush=True)

for num in range(2, n+1):
   temp = n_minus_one + n_minus_two
   if num < n:
       print(f"{temp}, ", end='', flush=True)
   else:
       print(f"{temp}", flush=True)
   n_minus_two = n_minus_one
   n_minus_one = temp

File: compRasp/fib.c
// A01732610 - Dafne Vania Peña Cortés

#include <stdio.h>

void fibonacci(int n) {
    int sig = 1;
    int curr = 0;
    int temp = 0;
    for (int x = 1; x <= n; x++) {
        printf("%d, ", curr);
        temp = curr;
        curr = sig;
        sig = sig + temp;
    }
    printf("%d", curr);
}

int main()
{
    printf("Los primeros 19 números de la serie Fibonacci son:\n");
    int n = 18;
    fibonacci(n);

    return 0;
}